docs/content/docs/try-flink-ml/python/quick-start.md

```diff
@@ -30,53 +30,56 @@ under the License.
 
 This document provides a quick introduction to using Flink ML. Readers of this
 document will be guided to submit a simple Flink job that trains a Machine
-Learning Model and use it to provide prediction service.
+Learning Model and uses it to provide prediction service.
 
 ## Prerequisites
 
-Python version (3.6, 3.7, or 3.8) is required for Flink ML. Please run the
-following command to make sure that it meets the requirements:
+This walkthrough assumes that you have some familiarity with Python, but you
+should be able to follow along even if you come from a different programming
+language.
 
-```shell
-$ python --version
-# the version printed here must be 3.6, 3.7 or 3.8
-```
+If you want to follow along, you will require a computer with:
+
+- Java 8
+- Maven 3
+- Python 3.6, 3.7 or 3.8
 
-## Installation of Flink ML Python SDK
+## Install Flink ML Python SDK
+
+{{< stable >}}
 
 Flink ML Python SDK is available in
 [PyPi](https://pypi.org/project/apache-flink-ml/) and can be installed as
 follows:
 
-{{< stable >}}
-
 ```bash
 $ python -m pip install apache-flink-ml=={{< version >}}
 ```
 
 {{< /stable >}} {{< unstable >}}
 
-```bash
-$ python -m pip install apache-flink-ml
-```
 
-{{< /unstable >}}
+Please walk through this [guideline]({{< ref "docs/development/building" >}}) to
```

Comment:
Would it be more straightforward to use `https://nightlies.apache.org/flink/flink-ml-docs-master/docs/development/building/#build-flink-ml-python-sdk` as the reference link?